.stylelintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+	"extends": "stylelint-config-wikimedia"
+}
```

Comment:
missing new line

Reply:
(Can we have a rule for that?)

Reply:
stylelint already support that, so we are fine with less file in the new line rule.